### ISSUE TITLE
[Segment] Placeholder segments do not play well with tabs

### ISSUE THREAD

**Issue by GammaGames** [fomantic/Fomantic-UI#1920]
# Bug Report
* A placeholder tab segment does not hide when not active
* A placeholder tab segment does not have the correct `display` value when active

## Steps to reproduce

1. Create tab with segments
2. Add placeholder to one of the segments
3. Call js `tab` function

## Expected result
Placeholder tab should work like a regular segment

## Actual result
Placeholder tab does not work like a regular segment

## Testcase
https://jsfiddle.net/eo7sut0g/

![image](https://user-images.githubusercontent.com/7832163/111848987-27f7e800-88d2-11eb-89d1-f852f38eddb1.png)
![image](https://user-images.githubusercontent.com/7832163/111848993-2d553280-88d2-11eb-8cbc-f4b0e3699f01.png)

## Version

2.8.7

**Comment by GammaGames** [comment]
This is my solution for now: https://jsfiddle.net/dn3o87y2/

```javascript
$('.menu .item').tab({
  onVisible: tab => {
    $(".ui.tab.segment").css("display", "none");
    let $segment = $(`.segment[data-tab="${tab}"]`);
    if($segment.hasClass("placeholder")) {
      $(`.segment[data-tab="${tab}"]`).css("display", "flex");
    }
    else {
      $(`.segment[data-tab="${tab}"]`).css("display", "");
    }
  }
});
```

**Comment by lubber-de** [comment]
Fixed by #1924

**Comment by GammaGames** [comment]
Adding some css would have been much easier than that absolute hack I put together
